spec/datetimeformat.html

```diff
@@ -364,17 +364,70 @@ <h1>FormatDateTimeToParts ( _dateTimeFormat_, _x_ )</h1>
     </emu-clause>
 
     <emu-clause id="sec-tolocaltime" aoid="ToLocalTime">
-      <h1>ToLocalTime ( _date_, _calendar_, _timeZone_ )</h1>
+      <h1>ToLocalTime ( _t_, _calendar_, _timeZone_ )</h1>
 
       <p>
-        When the ToLocalTime abstract operation is called with arguments _date_, _calendar_, and _timeZone_, the following steps are taken:
+        When the ToLocalTime abstract operation is called with arguments _t_, _calendar_, and _timeZone_, the following steps are taken:
       </p>
 
       <emu-alg>
-        1. Apply calendrical calculations on _date_ for the given _calendar_ and _timeZone_ to produce weekday, era, year, month, day, hour, minute, second, and inDST values. The calculations should use best available information about the specified _calendar_ and _timeZone_, including current and historical information about time zone offsets from UTC and daylight saving time rules. If the _calendar_ is `"gregory"`, then the calculations must match the algorithms specified in ES2020, <emu-xref href="#sec-overview-of-date-objects-and-definitions-of-abstract-operations"></emu-xref>.
-        1. Return a Record with fields [[weekday]], [[era]], [[year]], [[month]], [[day]], [[hour]], [[minute]], [[second]], and [[inDST]], each with the corresponding calculated value.
+        1. Assert: Type(_t_) is Number.
+        1. If _calendar_ is `"gregory"`,
+          1. Let _timeZoneOffset_ be the value calculated according to <emu-xref href="#sec-local-time-zone-adjustment">LocalTZA(_t_, *true*)</emu-xref> where the local time zone is replaced with timezone _timeZone_.
+          1. Let _tz_ be the time value _t_ + _timeZoneOffset_.
+          1. Return a record with fields calculated from _tz_ according to <emu-xref href="#table-datetimeformat-tolocaltime-record"></emu-xref>.
+        1. Else,
+          1. Return a record with the fields of Column 1 of <emu-xref href="#table-datetimeformat-tolocaltime-record"></emu-xref> calculated from _t_ for the given _calendar_ and _timeZone_. The calculations should use best available information about the specified _calendar_ and _timeZone_, including current and historical information about time zone offsets from UTC and daylight saving time rules.
       </emu-alg>
 
+      <emu-table id="table-datetimeformat-tolocaltime-record">
+        <emu-caption>Record returned by ToLocalTime</emu-caption>
+        <table class="real-table">
+          <thead>
+            <tr>
+              <th>Field</th>
+              <th>Value Calculation for Gregorian Calendar</th>
+            </tr>
+          </thead>
+          <tr>
+            <td>[[weekday]]</td>
+            <td>`WeekDay(tz)` specified in ES2020's <emu-xref href="#sec-week-day">Week Day</emu-xref></td>
+          </tr>
+          <tr>
+            <td>[[era]]</td>
+            <td></td>
```

Comment:
Era?

Reply:
I actually am a bit confused about what to put for era here and wanted to reach out, @sffc and @leobalter -- "era" is not discussed in 262, and [PartitionDateTimePattern](https://tc39.es/ecma402/#sec-partitiondatetimepattern) does not seem to expect a particular format. Should we just choose "AD" and "BC" and return based on the year? :/

Reply:
Yes, I would like Gregorian calculation to be very well defined.  FYI, the CLDR data that ICU uses to determine era is here:

https://github.com/unicode-org/cldr/blob/release-35-1/common/supplemental/supplementalData.xml#L4289

I think it is very reasonable to write that out here.

PartitionDateTimePattern is also really hand-wavy and needs some work.  @Ms2ger is working on that a bit in tc39/proposal-temporal#140.

Reply:
This string isn't exposed to the user, so I don't think it matters if you use AD/BC or 0/1 (or, for that matter, CE/BCE).